site/soc/software/charts/osh/openstack-tempest/tempest.yaml

```diff
@@ -0,0 +1,100 @@
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: openstack-tempest-soc
+  layeringDefinition:
+    abstract: false
+    layer: site
+    parentSelector:
+      name: tempest-global
+      component: tempest
+    actions:
+      - method: merge
+        path: .
+  storagePolicy: cleartext
+data:
+  wait:
+    timeout: 3000
```

Comment:
is 3000 sufficient to run full test?

Reply:
Mine finished, but I hadn't included any of the scenario tests as Flavio pointed out below. I increased it to 6000. Should this be enough?